resources/views/front-end/photo_album_list.blade.php

```diff
@@ -0,0 +1,130 @@
+@push('styles')
+
+@endpush
+@extends('layouts.app')
+
+@section('content')
+    <div class="container intro-container">
+        <div class="card">
+            <div class="card-body">
+                <button type="button" class="btn btn-outline-primary float-right" data-toggle="modal" data-target="#AddAlbumModal">{{trans('front-end/photo.addAlbum')}}</button>
+                <h2 class="card-title">Foto albums der Esac</h2>
+                Hier zijn fotos van klimweekenden, zomertochtjes en borrels te vinden!
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row d-flex align-items-stretch align-items-center">
+            @foreach ($photoAlbums as $album)
+                <div class="col-lg-4 col-md-6 d-flex flex-wrap">
+                    <div class="card w-100 position-relative">
+                        <a href="photoalbums/{!! $album->id !!}">
+                            <img class="card-img-top" src="{!! $album->thumbnail !!}" alt="Card image cap">
+                        </a>
+                        <span class="card-date position-absolute bg-light py-1 px-3 rounded">{{\Carbon\Carbon::parse($album->date)->format('d M Y')}}</span>
+                        <div class="card-body">
+                            <a href="photoalbums/{!! $album->id !!}">
+                                <h4 class="card-title">{!! $album->title !!}</h4>
+                                <p class="card-text text-body">{!! nl2br($album->description) !!}</p>
+                            </a>
+                        </div>
+                        <div class="card-footer bg-white p-3">
+                            <a class="btn btn-outline-primary" href="photoalbums/{!! $album->id !!}">{{trans('front-end/photo.show')}}</a>
+                        </div>
+                    </div>
+                </div>
+            @endforeach
+        </div>           
+    </div>
+    {{ $photoAlbums->links('front-end.pagination') }}
+
+    <div id="AddAlbumModal" class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
```

Comment:
Er moet nog een front end form validation komen op de lengte van de title en description. Nu krijg je een hele rare error terug als je titel of description te lang is (vanuit SQL). Dan kunnen we ook meteen checken of alle velden wel ingevuld zijn, *voordat* de images geprocessed worden.

Reply:
Deels opgelost in commit b476444
Enige wat nog niet klopt zijn de translations, ik kon met een (korte) google search niet vinden hoe ik dit kan aanpassen via javascript. Heb jij enige idee?

Reply:
Volgensmij is het veel makkelijker om gebruik te maken van HTML5 client-side validation ipv zelf in javascript iets te maken. Dan heb je ook geen problemen met engels/nederlands dat wordt dan door de browser afgehandeld.

https://www.the-art-of-web.com/html/html5-form-validation/

Reply:
Opgelost met HTML5 Validation in commit 2c00a1f